code/integration-tests/local-integration-tests/src/cross_chain_transfer.rs

```diff
@@ -890,7 +900,7 @@ fn sibling_trap_assets_works() {
 			RawOrigin::Root.into(),
 			any_asset,
 			remote,
-			Ratio::checked_from_integer::<u128>(1),
+			Some(rational_64!(1 / 1)),
```

Comment:
no need to have 64 in macro names.  you can just do into on tuple and unwrap

Reply:
I used the _64 suffix in case we have macros for other Rational* types in the future (Rational128/32/16/etc); it also makes it clearer what type is being constructed.

Reply:
The day when macros are namespaced will be a good day :)